src/components/modals/ClassInfo/ClassInfo.tsx

```diff
@@ -354,6 +355,16 @@ export default function ClassInfo({
                 </Box>
             )}
             <Typography pt={2}>{_class.activity.description}</Typography>
+            {user === undefined && !isInThePast && (
```

Comment:
Nice feedback!

Have you considered showing the alert above the class image, to make it more discoverable? Currently it's placed where the buttons would be, but a new user might not be aware of where those usually are.

Reply:
It might still work to have everything at the bottom though, I feel like the info is mostly for users that have randomly been logged out and tries to figure out why they cannot add something to their agenda, not new users exploring.

Reply:
What do you think about this?

Reply:
Soo, I thought a bit about this: I think it makes sense to make the "you need to log in" and "you need to set up a chain user" alerts sticky. This way they are very visible, and it is obvious that you can scroll, which helps new users.

I also thought about making everything sticky, so that users more easily can find the "book now" and "add to agenda" buttons. However, I think that can be a bit daunting if they have the inactive booking alert. So I think just sticking the login/user alerts is wise. 

Implemented in: https://github.com/mathiazom/rezervo-web/pull/100/commits/8d48c8eeee449a9e521f6b21d8f4171973f90b08

Do you agree, or do you have other suggestions?

Reply:
https://github.com/mathiazom/rezervo-web/assets/26925695/a372c475-3606-4be5-8e0f-31b7e74f969f

Reply:
This is mostly personal preference, but I'm not a huge fan of using sticky here (kind of reminds me of the infamous cookie-banners 🙃 )

It's also still at the bottom, so not as discoverable as having it below the image.

Reply:
Sure, I think it looks fine above the image too! See latest comment for the latest design. ✅